apis/python/src/tiledb/vector_search/ivf_flat_index.py

```diff
@@ -153,7 +156,21 @@ def query_internal(
             wether to use the nuv query implementation. Default: False
         mode: Mode
             If provided the query will be executed using TileDB cloud taskgraphs.
-            For distributed execution you can use REALTIME or BATCH mode
+            For distributed execution you can use REALTIME or BATCH mode. 
+            For local execution you can use LOCAL mode.
+        resource_class: 
+            The name of the resource class to use ("standard" or "large"). Resource classes define maximum
+            limits for cpu and memory usage. Can only be used in REALTIME or BATCH mode.
+            Cannot be used alongside resources.
+            In REALTIME or BATCH mode if neither resource_class nor resources are provided,
+            we default to the "large" resource class.
+        resources:
+            A specification for the amount of resources to use when executing using TileDB cloud 
+            taskgraphs, of the form: {"cpu": "6", "memory": "12Gi", "gpu": 1}. Can only be used 
+            in REALTIME or BATCH mode. Cannot be used alongside resource_class.
+        num_partitions: int
```

Comment:
Duplicate documentation for `num_partitions`

Reply:
Thanks, good catch, removed.